charts/stable/standardnotes-server/values.yaml

```diff
@@ -0,0 +1,285 @@
+image:
+  pullPolicy: IfNotPresent
+  repository: standardnotes/server
+  tag: latest@sha256:cbdc5e3e6d7212eeaec86898e6ba944b1678349262d3da7215a246d832471391
+localstackImage:
+  pullPolicy: IfNotPresent
+  repository: localstack/localstack
+  tag: 3.0@sha256:73698e485240939490134aadd7e429ac87ff068cd5ad09f5de8ccb76727c13e1
```

Comment:
Is there a full semver tag maybe?

Reply:
Yes, 3.0.2 has the same checksum. Added it.
https://hub.docker.com/layers/localstack/localstack/3.0.2/images/sha256-73698e485240939490134aadd7e429ac87ff068cd5ad09f5de8ccb76727c13e1?context=explore